crates/biome_css_analyze/tests/specs/nursery/noUnknownPseudoClassSelector/invalid.css

```diff
@@ -0,0 +1,15 @@
+a:unknown { }
+a:Unknown { }
+a:uNkNoWn { }
+a:UNKNOWN { }
+a:pseudo-class { }
+body:not(div):noot(span) {}
+a:unknown::before { }
+a,\nb > .foo:error { }
```

Comment:
```suggestion
a,
b > .foo:error { }
```

Reply:
I fixed it in https://github.com/biomejs/biome/pull/3034/commits/31441057cb0fca66508afb155054ca170fe32894 and update snapshots